ts/browser.ts

```diff
@@ -108,41 +114,50 @@ class Browser {
       protocol: "web-greeter:",
     });
     //console.log({ theme_url, url: new URL(theme_url) });
-    this.win.loadURL(`${theme_url}`);
-    this.win.setBackgroundColor("#000000");
-
-    this.win.webContents.on("before-input-event", (_event, input) => {
-      let value = nody_greeter.config.features.backlight.value;
-      if (input.type == "keyUp") return;
-      if (input.code == "BrightnessDown") {
-        Brightness.dec_brightness(value);
-      } else if (input.code == "BrightnessUp") {
-        Brightness.inc_brightness(value);
-      }
-    });
+    for (const w of this.windows) {
+      w.window.loadURL(`${theme_url}`);
+      w.window.setBackgroundColor("#000000");
+
+      w.window.webContents.on("before-input-event", (_event, input) => {
+        let value = nody_greeter.config.features.backlight.value;
+        if (input.type == "keyUp") return;
+        if (input.code == "BrightnessDown") {
+          Brightness.dec_brightness(value);
+        } else if (input.code == "BrightnessUp") {
+          Brightness.inc_brightness(value);
+        }
+      });
+    }
 
     logger.debug("Theme loaded");
   }
 
-  create_window(): BrowserWindow {
+  create_window(): NodyWindow[] {
```

Comment:
```suggestion
  create_windows(): NodyWindow[] {
```

Change name to be consistent with its return type.